src/subPages/SimilarProteins/Table/index.tsx

```diff
@@ -108,33 +108,37 @@ const SimilarProteinTable = ({
         <PageSizeSelector />
         <Exporter>
           <div className={css('menu-grid')}>
-            <label htmlFor="fasta">FASTA</label>
             <AllProteinDownload
               description={description}
               ida={ida}
               db={db}
               count={payload.count}
               fileType="fasta"
             />
-            <label htmlFor="json">JSON</label>
             <AllProteinDownload
               description={description}
               ida={ida}
               db={db}
               count={payload.count}
               fileType="json"
             />
-            <label htmlFor="tsv">TSV</label>
             <AllProteinDownload
               description={description}
               ida={ida}
               db={db}
               count={payload.count}
               fileType="tsv"
             />
-            <label htmlFor="api">API</label>
-            <APIViewButton url={getAPIURLForSimilarProteins(api, ida, db)} />
+            <ExternalExportButton
+              type={'api'}
+              url={getAPIURLForSimilarProteins(api, ida, db)}
+            />
           </div>
+          <ExternalExportButton
+            search={search}
+            type={'scriptgen'}
+            subpath={descriptionToPath(description)}
+          />
         </Exporter>
```

Comment:
```suggestion
            <ExternalExportButton
              type={'api'}
              url={getAPIURLForSimilarProteins(api, ida, db)}
            />
            <ExternalExportButton
              search={search}
              type={'scriptgen'}
              subpath={descriptionToPath(description)}
            />
          </div>
        </Exporter>
```

Move second `ExternalExportButton` within the same group.